10-workload.md

```diff
@@ -54,8 +54,6 @@ The cluster now has an [Traefik configured with a TLS certificate](./08-secret-m
    exit
    ```
 
-   > :beetle: If you are running a version of kubectl less than 1.23 you cannot perform this step.  The Azure Policy for Kubernetes assignments that are deployed with this implementation requires both setting limits and also a read-only root filesystem. This isn't possible with kubectl 1.22 and lower. It's okay to skip this step if in that situation.
```

Comment:
why is this no longer valid? Are we requesting to install a more recent version now?

Reply:
the requirements say at least 1.24, and 1.23 is not supported to manage a 1.25 cluster. So no reason to talk about non-supported kubectl problems.